epoch/api/chain_ws_api.md

```diff
@@ -411,32 +403,44 @@ Response:
 ```
 
 ## Data types
-### Header structure
+### Block structure
+A block can be either a key block or a micro block. [The key block](#key-block-structure) does not include any transactions. The micro block consists of a [header](#micro-block-header-structure) and a list of transactions.
```

Comment:
list can be empty, right?

Reply:
Empty micro blocks (without txs) are not produced.

Reply:
This seems [not guaranteed](https://github.com/aeternity/epoch/pull/1293/files#diff-2e95a69d7723aebad303814cb3f38e7eR142)?

Reply:
I [see now](https://github.com/aeternity/epoch/pull/1293/files#diff-4c1e0a7bf608a6ef03b1496e6218f75cR77).